apps/condo/domains/ticket/access/TicketFile.js

```diff
@@ -0,0 +1,66 @@
+
+/**
+ * Generated by `createschema ticket.TicketFile 'organization:Text;file?:File;ticket?:Relationship:Ticket:SET_NULL;'`
+ */
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+ 
+
+async function canReadTicketFiles ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) {
+        return {}
+    }
+    return {
+        OR: [{  
```

Comment:
Why you need to use OR?

Reply:
Thx - it's a good point. Also, make corrections to the property access